kivy/uix/textinput.py

```diff
@@ -2400,7 +2401,8 @@ def _key_down(self, key, repeat=False):
             if cursor != self.cursor:
                 self.do_backspace(mode='del')
         elif internal_action == 'backspace':
-            self.do_backspace()
+            if not self._is_textedit:#检查是否是输入法
```

Comment:
```suggestion
            if not self._is_textedit:
```
comment seems redundant, and if needed, should be in english.